config/optional-features.json

```diff
@@ -1,4 +1,11 @@
 {
+<<<<<<< HEAD
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('<' (code 60)): was expecting double-quote to start field name](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)